modules/engines/engine-xsjob/src/main/java/com/sap/xsk/xsjob/ds/scheduler/handler/XSKJobHandler.java

```diff
@@ -24,7 +24,8 @@
 
 public class XSKJobHandler implements Job {
 
-  private static final String XSJOB_HANDLER = "xsjob/wrappers/handler.xsjs";
+  private static final String XSJOB_HANDLER               = "xsjob/wrappers/handler.xsjs";
+  private static final String XSJOB_EXCEPTION_DESCRIPTION = "jobDescription";
```

Comment:
Move the constant to the `IXSKJobCoreService` and rename it to `XSK_JOB_DESCRIPTION`